BritishGeologicalSurvey/etlhelper
# Commit after fetching data

### Summary

This merge request makes `etlhelper` call `conn.commit()` after fetching data.  This closes the transaction that was used to run the SQL query, preventing it being reused for future queries with unexpected consequences.

Closes #93

### To test

The new unit tests demonstrate the new functionality.  To see for yourself, run `select now() as result` repeatedly against a PostgreSQL database.  Prior to this change, the same value was always returned.

+ [x] Confirm tests pass
